### ISSUE TITLE
console.log in JavaScript filtering

### ISSUE THREAD

**Issue by QuentinN42** [redpanda-data/console#1472]
Hello,

In the `JavaScript filtering`, I can't use `console.log(...)` to debug my filter.

It can be really useful as sometime I don't really know what JavaScript object I'm manipulating...

**Comment by weeco** [comment]
I agree the UX should be improved. Writing slightly more complex filters is hard to debug in Console. Sorry for that, unsure when we can get to this though.
